analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.InvocationAttribute.cs

```diff
@@ -200,14 +200,14 @@ public void Test()
         validator.TagStates("End").Should().SatisfyRespectively(
             x =>
             {
-                x[validator.Symbol("ObjectField")].HasConstraint<ObjectConstraint>().Should().BeFalse();
+                x[validator.Symbol("ObjectField")].Should().BeNull();
```

Comment:
All assertions in this UT should use `.Should().HaveNoConstraint()` instead of null, and `Should.HaveOnlyConstraints(.....)` for the rest